Jellyfin.Plugin.KodiSyncQueue/Data/DbRepo.cs

```diff
@@ -6,411 +6,201 @@
         }
 
-        //public List<string> GetFolders(long dtl, int status, string userId, bool movies, bool tvshows, bool music, bool musicvideos, bool boxsets)
-        //{
-        //    using (var trn = DB.BeginTrans())
-        //    {
-        //        var result = new List<string>();
-        //        List<FolderRec> final = new List<FolderRec>();
-        //        //
-        //        var flds = folders.Find(x => x.LastModified > dtl && 
-        //                                     x.Status == status && 
-        //                                     x.UserId == userId)
-        //                          .Where(x =>
-        //                          {
-        //                              switch (x.MediaType)
-        //                              {
-        //                                  case "movies":
-        //                                      if (movies) { return true; } else { return false; }
-        //                                  case "tvshows":
-        //                                      if (tvshows) { return true; } else { return false; }
-        //                                  case "music":
-        //                                      if (music) { return true; } else { return false; }
-        //                                  case "musicvideos":
-        //                                      if (musicvideos) { return true; } else { return false; }
-        //                                  case "boxsets":
-        //                                      if (boxsets) { return true; } else { return false; }
-        //                              }
-        //                              return false;
-        //                          })
-        //                          .ToList();
-
-        //        //if (movies) { final.AddRange(flds.Where(x => x.MediaType == "movies").ToList()); }
-        //        //if (tvshows) { final.AddRange(flds.Where(x => x.MediaType == "tvshows").ToList()); }
-        //        //if (music) { final.AddRange(flds.Where(x => x.MediaType == "music").ToList()); }
-        //        //if (musicvideos) { final.AddRange(flds.Where(x => x.MediaType == "musicvideos").ToList()); }
-        //        //if (boxsets) { final.AddRange(flds.Where(x => x.MediaType == "boxsets").ToList()); }
-
-        //        //final.ForEach(x =>
-        //        flds.ForEach(x =>
-        //        {
-        //            if (result.Where(i => i == x.ItemId).FirstOrDefault() == null)
-        //            {
-        //                result.Add(x.ItemId);
-        //            }
-        //        });
-        //        return result;
-        //    }
-        //}
-
-        public List<UserJson> GetUserInfos(long dtl, string userId, bool movies, bool tvshows, bool music, bool musicvideos, bool boxsets)
+        public List<UserJson> GetUserInfos(long dtl, string userId, IReadOnlyCollection<MediaType> filters)
```

Comment:
why does `GetUserInfos` accept `filters`?

Reply:
It uses it?

Reply:
But why should it? Isn't it getting info on what users are available?..

Reply:
Users can configure which libraries to sync in Kodi

Reply:
I might not be understanding what this function does then...